v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -167,10 +169,17 @@ func AddSubcommand(app *clir.Cli, w io.Writer) error {
 
 		if !flags.noGen {
 			self := os.Args[0]
+			var env []string
+			if flags.obfuscate {
```

Comment:
This needs to check for the exact type vs a truth value.

Reply:
That's a boolean so we're good 👍

Reply:
Nice, nice